lib/stormpy/__init__.py

```diff
@@ -300,7 +300,9 @@ def check_model_sparse(model, property, only_initial_states=False, extract_sched
         if force_fully_observable:
             # Note that casting a model to a fully observable model wont work with python/pybind, so we actually have other access points
             if model.supports_parameters:
-                raise NotImplementedError("")
+                raise NotImplementedError("Model checking of partially observable models is not supported for parametric models.")
+            elif model.supports_uncertainty:
+                raise NotImplementedError("Model checking of partially observable models is not supported for interval models.")
```

Comment:
but this is if force_fully_observable is set to true, which is basically casting the POMDP to an MDP. It is debatable whether this part of the API is pretty though

Reply:
That is a good point. To keep things a bit clear, a opened a new issue, see https://github.com/moves-rwth/stormpy/issues/151. My proposal is to keep it here for now but change it in a separate PR.

Reply:
ok